ledger/migration/src/lib.rs

```diff
@@ -89,6 +90,10 @@ pub fn migrate(ledger_db_path: impl AsRef<Path>, logger: &Logger) {
                     "Ledger db migrating from version 2020_07_07 to 2022_02_22..."
                 );
                 MintConfigStore::create(&env).expect("Failed creating MintConfigStore");
+                MintTxStore::create(&env).expect("Failed creating MintTxStore");
+
+                backfill_empty_mint_stores(&env, logger)
+                    .expect("Failed backfilling empty mint stores");
```

Comment:
What about DBs at version `2022_02_22` that don't have `MintTxStore`s?

Reply:
Those do not exist in the wild, since that is the version that is going to be introducing all the minting-related changes (this version only exists in the `feature/minting` branch)